postgres-appliance/scripts/configure_spilo.py

```diff
@@ -719,6 +719,8 @@ def write_wale_environment(placeholders, prefix, overwrite):
             elif not wale_endpoint:
                 wale_endpoint = aws_endpoint.replace('://', '+path://')
             wale.update(WALE_S3_ENDPOINT=wale_endpoint, AWS_ENDPOINT=aws_endpoint, WALG_DISABLE_S3_SSE='true')
+            if wale.get('USE_WALG_BACKUP') and wale.get('USE_WALG_BACKUP') == 'true':
```

Comment:
We want to be sure that aws_region is actually defined:
```suggestion
            if aws_region and wale.get('USE_WALG_BACKUP') == 'true':
```